plugins/woocommerce/client/legacy/css/admin.scss

```diff
@@ -7955,3 +7749,24 @@ table.bar_chart {
 		}
 	}
 }
+
+#wp-content-media-buttons {
```

Comment:
Changes in this file starts from this line. Other changes were caused by the linter.